src/Core/PluginCollection.php

```diff
@@ -131,7 +131,7 @@ public function get($name)
      */
     public function next()
     {
-        $this->position += 1;
+        ++$this->position;
```

Comment:
The old form is easier to read in my opinion.

Reply:
We use the prefix version? I didn't even know that was a thing in PHP.

Reply:
We do use the suffix form, but generally try to avoid the prefix form as it uncommon.

Reply:
I found 4 other places we are using prefix incrementing. I'll normalize all of the prefix usage after merging this.

Reply:
Perfect, thank you, Mark.